randbp/seed.go

```diff
@@ -3,18 +3,39 @@ package randbp
 import (
 	"crypto/rand"
 	"encoding/binary"
+	"fmt"
+	"io"
+	"os"
 	"time"
 )
 
-// GetSeed returns a seed for pseudo-random generator.
+// Define as variables so they can be mocked in unit tests.
+var (
+	cryptoReader               = rand.Read
```

Comment:
Calling this “cryptoReader” seems misleading since it’s using rand rather than crypto

Reply:
it's `crypto/rand` (there's also `math/rand`)